ui/appui-react/src/appui-react/childwindow/CopyStyles.ts

```diff
@@ -22,12 +24,19 @@ export async function copyStyles(
       clonedNode.href
     ) {
       promises.push(
-        new Promise((resolve, reject) => {
+        new Promise((resolve) => {
           clonedNode.onload = () => {
             resolve();
           };
-          clonedNode.onerror = (error) => {
-            reject(error);
+          clonedNode.onerror = () => {
+            Logger.logWarning(
```

Comment:
I think this should at least show a `console.error`

Reply:
It will, depending on how the logger is configured. We could increase `LogLevel` to `Error` (instead of a `Warning`) though. We try to avoid direct `console` calls.

Reply:
Changed to `logError()`